testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -26,6 +26,8 @@
     SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN(SUPPORTS_PREDICATE_PUSHDOWN),
     SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN_WITH_LIKE(SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN),
 
+    SUPPORTS_DYNAMIC_FILTER_PUSHDOWN(false),
```

Comment:
What does this affect? Shouldn't it be true by default as we added support in general JDBC connectors?

Reply:
Currently it affects which JDBC connects will run DF tests, it's true by default in `BaseJdbcConnectorTest#hasBehavior`. It doesn't make sense to set it to true here until we generalise DF tests to all connectors and add those tests in `BaseConnectorTest`.